site/_config.yml

```diff
@@ -2,6 +2,9 @@ markdown: kramdown
 highlighter: pygments
 permalink: /news/:year/:month/:day/:title/
 excerpt_separator: ""
+kramdown:
+  input: GFM
+  hard_wrap: false
```

Comment:
I believe this has deleterious effects on the markup of the site. Please submit this in another pull request.